tools/tslint-rules/disallowSpecificEnvironmentImportRule.ts

```diff
@@ -0,0 +1,45 @@
+import { ImportDeclaration, SourceFile } from 'typescript';
+import { RuleFailure, Rules, RuleWalker, IRuleMetadata } from 'tslint';
+
+export class Rule extends Rules.AbstractRule {
+  static readonly metadata: IRuleMetadata = {
+    ruleName: 'disallow-specific-environment-import',
+    type: 'maintainability',
+    description: `Ensures that no imports contain specific environment.ts file (as it's handled by Angular/webpack).`,
+    options: null,
+    optionsDescription: 'Not configurable',
+    rationale: `Explicitly importing 'environment.mock.ts', 'environment.prod.ts' etc. is a no-go.`,
+    typescriptOnly: false,
+  };
+
+  static readonly FAILURE_STRING = `Do not import from specific environment file as it's handled by Angular/webpack. Remove the suffix ('.mock' or '.prod').
+*** Please note: Environment import can be simplified as: "import { environment } from '@drb-environment';"`;
```

Comment:
Well we don't have @drd-environment in Kirby? 😄 

We should properly change this error string.

Reply:
I agree... will do!